### ISSUE TITLE
Some commands changed

### ISSUE THREAD

**Issue by niranjankm** [DawesLab/Instruments#4]
Thank you for the excellent project. Please have a look at the following programmer manual from Tektronix. Some commands have changed (some changed in case (lowercase)). For my new oscilloscope I had to change them so that the TekScope.py script worked finally.

http://www.tek.com/oscilloscope/tds1000-manual

**Comment by amcdawes** [comment]
Interesting, can you clarify which commands you had to modify? I have not had issues with the case of the commands sent, and that manual even says "Enter commands in upper or lower case." on pg 2-4. I'm happy to look into it further, but without knowing what worked and what didn't, I don't have much to go on. Thank you, and I'm glad you found the code useful.

**Comment by niranjankm** [comment]
Hi, I am really sorry, I hurried in reading the manual and just jumped into the commands. Now I rechecked it all, I misunderstood the full command (which manual showed with small letters in end) and its abbreviation. Before I proceed let me confess one more thing that I am doing it with my Tek TBS1102B oscilloscope (TBS1000 series). Though it is not a TDS series (which your code is actually meant for), as the programmer manual is same for both TBS* and TDS*, so I thought the basic commands are same for both series and hence it will work. But there are some exceptions, wherever mentioned in the manual, like return of "SELECT?" query is different for some series. Now, I could think, in the beginning TekScope.py did not work for me only because of what "SELECT?" command returned for my oscilloscope. This I figured out and fixed but in those tries the scope sometimes ended up with error codes and was not communicating, which I wrongly thought as due to differences is commands with that of the manual. Thank you for enlightening.

Your TekScopeGUI.py and TekScopeQt.py (did not try TekScopeQt5.py yet) script works even for TBS1000 series. TekScope.py also works for TBS1000 series if the string returned for "SELECT?" is handled appropriately.
